kubernetes/loculus/values.yaml

```diff
@@ -1377,10 +1377,12 @@ defaultOrganisms:
           sequence: "[[URL:https://corneliusroemer.github.io/seqs/artefacts/sars-cov-2/ORF9b.fasta]]"
         - name: "S"
           sequence: "[[URL:https://corneliusroemer.github.io/seqs/artefacts/sars-cov-2/S.fasta]]"
-  not-aligned-organism:
```

Comment:
I don't know if we want to lose the not-aligned organism - that's an important case

Reply:
I can't judge how important this one is. Should I bring it back?

Reply:
I think that it'd be good to keep it for now.

Reply:
Done